src/app/static/src/tests/components/runWorkflow/runWorkflowCreate.test.ts

```diff
@@ -28,18 +67,86 @@ describe(`runWorkflowCreate`, () => {
         expect(wrapper.emitted("create").length).toBe(1)
     })
 
-    it(`can emit run navigation step`, async() => {
+    it(`can emit run navigation step`, async (done) => {
```

Comment:
Let's use re-run here, "run" is generic and applies to cloning and re-running.
```suggestion
    it(`can emit rerun navigation step`, async (done) => {
```